server/src/lfortran-accessors.ts

```diff
@@ -282,6 +282,9 @@ export class LFortranCLIAccessor implements LFortranAccessor {
     const start: number = performance.now();
 
     const flags = ["--show-document-symbols", "--continue-compilation"];
+    if (settings.compiler.flags !== "") {
+      flags.push(settings.compiler.flags);
```

Comment:
Once the flags are made an array, `flags` will need to be made mutable and this will need to be changed to `flags = flags.concat(settings.compiler.flags);`.

Reply:
Resolved in favor of https://github.com/Pranavchiku/lfortran-lsp/pull/1